### PR TITLE
Add the new Set component to the Router docs

### PR DESCRIPTION
**Pull request by jtoar**
Documents the new `Set` component introduced in https://github.com/redwoodjs/redwood/pull/1898.

### CODE REVIEW

File: packages/router/README.md [hunk 1]
```diff
@@ -57,6 +57,72 @@ they will be redirected to the route passed as the `unauthenticated` prop and th
...
 
+## Sets of Routes
+
+You can group routes into sets using the `Set` component. `Set` allows you to wrap a set of Routes in another component or component(s)—usually a Context, a Layout, or both:
+
+```js
+// Routes.js
+
+import { Router, Route, Set } from '@redwoodjs/router'
+import BlogContext from 'src/components/BlogContext'
```
Comment (Tobbe) [inline packages/router/README.md:68]:
```suggestion
import BlogContext from 'src/contexts/BlogContext'
```

/components is already too crowded as it is. I like to put my contexts in a folder of their own

File: packages/router/README.md [hunk 2]
```diff
@@ -57,6 +57,72 @@ they will be redirected to the route passed as the `unauthenticated` prop and th
...
+export default Routes
+```
+
+The `wrap` prop accepts a single component or an array of components. Components are rendered in the same order they're passed, so in the exmaple above, Set expands to:
+
+```js
+<BlogContext>
+  <BlogLayout>
+    <Route path="/" page={HomePage} name="home" />
+    ...
```
Comment (Tobbe) [inline packages/router/README.md:93]:
```suggestion
    // ...
```
To keep with the "syntax" in the tutorial

File: packages/router/README.md [hunk 3]
```diff
@@ -57,6 +57,72 @@ they will be redirected to the route passed as the `unauthenticated` prop and th
...
+
+Conceptually, this fits with how we think about Context and Layouts as things that wrap Pages and contain content that’s outside the scope of the Pages themselves. Crucially, since they're higher in the tree, `BlogContext` and `BlogLayout` won't rerender across Pages in the same Set.
+
+There's a lot of flexibility here. You can even nest `Sets` to great effect:
+
+```js
+// Routes.js
+
+import { Router, Route, Set, Private } from '@redwoodjs/router'
+import BlogContext from 'src/components/BlogContext'
```
Comment (Tobbe) [inline packages/router/README.md:106]:
```suggestion
import BlogContext from 'src/contexts/BlogContext'
```
as above

File: packages/router/README.md [hunk 4]
```diff
@@ -57,6 +57,72 @@ they will be redirected to the route passed as the `unauthenticated` prop and th
 Redwood uses the `useAuth` hook under the hood to determine if the user is authenticated.
 Read more about authentication in redwood [here](https://redwoodjs.com/tutorial/authentication).
 
+## Sets of Routes
+
+You can group routes into sets using the `Set` component. `Set` allows you to wrap a set of Routes in another component or component(s)—usually a Context, a Layout, or both:
```
Comment (Tobbe) [inline packages/router/README.md:62]: Just "component(s)" or "component or components". With the current wording the (s) isn't really optional

**Review by Tobbe — APPROVED** [review]
Minor comments. Change if you want. Ignore if you want.

File: packages/router/README.md (hunk 4 above)
Comment (jtoar) [reply packages/router/README.md:62]: Nice totally missed that

**Review by thedavidprice — APPROVED** [review]

**Pull request merged**